examples/securityhub/actiontarget.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: securityhub.aws.upbound.io/v1beta1
+kind: ActionTarget
+metadata:
+  annotations:
+    meta.upbound.io/example-id: securityhub/v1beta1/actiontarget
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    description: This is custom action sends selected findings to chat
+    name: Send
+    region: us-west-1
+---
+apiVersion: securityhub.aws.upbound.io/v1beta1
```

Comment:
There are bunch of `Account` examples that don't seem to be referenced by other resources. If they're not needed, can we remove them?

Reply:
@muvaf Unfortunately without Account this resource doesn't work. There is an error from this case:
cannot run refresh: refresh failed: Error reading Security Hub custom action targets to find arn:aws:securityhub:us-west-1:153891904029:action/custom/example: InvalidAccessException: Account 153891904029 is not subscribed to AWS Security Hub

Reply:
Hmm so they don't reference directly but the resource has to be created with the same ProviderConfig that subscribed as account to the Security Hub. In that case, it would work if we had a single example file for the whole folder that has 1 Account and all other examples, right? Would it make sense to try it?

Reply:
Do you mean creating a new file that includes all resources in one instance?

Reply:
Yes, 1 instance of every kind in this folder. They could go to different files in the same folder and we can trigger Uptest for the whole folder as well. The goal is essentially to reduce the amount of duplication of examples in a given group.